### ISSUE TITLE
Remove fixed requests requirement

### ISSUE THREAD

**Issue by chripede** [forcedotcom/SalesforcePy#54]
Your have fixed your Requests version to == 2.20.0
That is bad practice and conflicts with a lot of other packages that we are using that are also using Requests.

```
The conflict is caused by:
    The user requested requests==2.25.0
    b2sdk 1.0.2 depends on requests>=2.9.1
...
    salesforcepy 2.0.0 depends on requests==2.20.0
```

**Comment by pepefloyd** [comment]
@chripede  Feel free to submit a backwards compatible PR.

**Comment by caffalaughrey** [comment]
Fixed in #55
